README.md

```diff
@@ -214,23 +214,17 @@ You'll see the `plugins.cookie` section, which is loading `examples/plugin/nemo-
 'use strict';
 
 module.exports = {
-  "setup": function (nemo, callback) {
+  "setup"(nemo, callback) {
```

Comment:
@skratchdot Ah! so only the export default doesn't need the use of colon? - I wondering how the tests passed and didnt show any lint errors